content/introduction-and-setup/walkthrough/creating-a-project/index.md

```diff
@@ -0,0 +1,85 @@
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+Following the “Hello World” trend, let’s create a new Visual Studio project.
+
+1. Create a new folder to hold your C# practice files. Since you will be creating lots of small projects as you move through this course, we suggest that you also add sub-folders with names corresponding to the related chapters and projects. Something like `csharp-practice/chapter-name/project-name`.
+1. In Visual Studio, from the project opener window, select the option to create a *New Visual Studio project*.
+1. You next need to choose what project template to use. For this first project (and those in the next several lessons), select the *.NET Core Console Application* option for C#.
+   1. **Mac Users**:
+      1. You can find this by selecting *Web* and *Console* options in the left menu.
+      1. Select the *App* subdirectory.
+      1. Select *Console Application* from the *General* menu option in the center menu pane.
+   1. **Windows Users**:
+      1. Select *Console App*.
+      1. If you can’t find it easily, you can search “Console App”.
+1. Select *.NET 6.0* as your Target Framework.
+1. Then, give your new project a name. Following C# Naming Conventions, call your project `HelloWorld`. The solution name will be the same. Choose where you want this project to be saved, ideally somewhere inside the directory you created in Step 1 of this tutorial.
+   1. **Mac Users**: Pick the options for “Use git for version control.” and “Create a .gitignore file to ignore inessential files”.
+1. Visual Studio will now create a **Solution** to hold your console **Project**. The project is your current app. The project contains all of the files that app needs to build and run. The solution is a workspace that combines multiple projects, which are usually related to each other. The solution file type is **.sln**. Once created, Visual Studio opens a new project window that displays your `Program.cs` file.
+1. To see the other files inside this solution, you will need to view either the Solution (Mac Users) or the Solution Explorer (Windows Users). You’ll see the project file tree containing a file called `Program.cs` in a pane called Solution Explorer.
+1. You are new to C# and we’ll go over the syntax present in Program.cs in time. For now, can you guess what line 2 accomplishes?
+
+   {{< highlight csharp "linenos=true" >}}
+      // See https://aka.ms/new-console-template for more information
+      Console.WriteLine("Hello, World!");
+   {{< / highlight >}}
+  
+   Click on the *Run* button (a triangle button located above the `Program.cs` panel) indicated in one of the images above to run the project and see the output.
+
```

Comment:
On the mac it is located above the solution pane.  Also the A Tour of Your New IDE for mac link gives a nice walkthrough of Hello World.  Sadly the Windows - A Tour of Your New IDE does not.  Option to rework this page As a tour and creating your first project page.

Reply:
Done!